conftest.py

```diff
@@ -0,0 +1,42 @@
+"""Place Pytest fixtures to be used across all tests here."""
+import pytest
+import ray
+
+
+@pytest.fixture(scope='module')
+def ray_local_test_fixture():
```

Comment:
can you give us some info on why we would use the local version versus the non-local version?